core/renderers/zelos/drawer.js

```diff
@@ -13,39 +13,40 @@
 goog.module('Blockly.zelos.Drawer');
 goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.blockRendering');
-goog.require('Blockly.blockRendering.Drawer');
-goog.require('Blockly.utils.object');
-goog.require('Blockly.utils.svgPaths');
-goog.require('Blockly.zelos.RenderInfo');
-
-goog.requireType('Blockly.blockRendering.Row');
-goog.requireType('Blockly.BlockSvg');
-goog.requireType('Blockly.zelos.PathObject');
+const BaseDrawer = goog.require('Blockly.blockRendering.Drawer');
```

Comment:
Nit: I'm not sure the style guide approves ("Base" is not a component of the `Blockly.blockRendering.Drawer`)—but `blockRenderingDrawer` doesn't seem like an improvment.

Reply:
Based on offline discussion, I will leave "Base" for this conversion.